fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -1949,12 +1948,15 @@ class FMCMainDisplay extends BaseAirliners {
     /**
      * Any tropopause altitude up to 60,000 ft is able to be entered
      * @param {string} tropo Format: NNNN or NNNNN Leading 0’s must be included. Entry is rounded to the nearest 10 ft
+     * @param {number} forPlan the flight plan index to set tropopause for
      * @return {boolean} Whether tropopause could be set or not
      */
-    tryUpdateTropo(tropo) {
+    tryUpdateTropo(tropo, forPlan) {
+        const plan = this.flightPlan(forPlan, false);
+
         if (tropo === FMCMainDisplay.clrValue) {
-            if (this.tropo) {
-                this.tropo = undefined;
+            if (plan.performanceData.tropopause) {
+                this.currFlightPlanService.setPerformanceData('tropopause', undefined, forPlan);
```

Comment:
Check whether you want an unset tropopause to be indicated by `undefined` or `null`. You indicate it's `number | null` in the performance data but set it to `undefined` here.

Reply:
We need strict mode... Done.